server/core/youtube-api.js

```diff
@@ -43,7 +43,18 @@ module.exports = class YoutubeApi {
 	 * @returns {Promise}
 	 */
 	getRelatedVideos(youtubeId) {
-		let youTubeApiRequestUrl = `${URL_BASE}search?type=video&relatedToVideoId=${youtubeId}&part=snippet&key=${this.apiKey}`;
+		let youTubeApiRequestUrl = `${URL_BASE}search?type=video&relatedToVideoId=${youtubeId}&part=snippet&videoEmbeddable=true&key=${this.apiKey}`;
+		return this.simpleGetRequest(youTubeApiRequestUrl);
+	}
+
+	/**
+	 * Performs youtube search.
+	 * ref: https://developers.google.com/youtube/v3/docs/search/list
+	 * @param  {String} qs Query string
+	 * @return {Promise}
+	 */
+	search(qs) {
+		const youTubeApiRequestUrl = `${URL_BASE}search?part=snippet&q=${qs}&type=video&videoEmbeddable=true&key=${this.apiKey}`;

```

Comment:
We could set `maxResults` (https://developers.google.com/youtube/v3/docs/search/list#maxResults) to something larger, as it defaults to `5` and we get very few autocomplete results. Ideally we could have a `more` link below the search results that loads the next page, but it might be too much of a hassle.